src/reducers/accounts.js

```diff
@@ -20,11 +20,14 @@ import {
   MKR_APPROVE_SUCCESS,
   IOU_APPROVE_SUCCESS
 } from './sharedProxyConstants';
-import { MAX_UINT_ETH_BN } from '../utils/ethereum';
+import { MAX_UINT_ETH_BN, getUrlParam } from '../utils/ethereum';
 import { MKR } from '../chain/maker';
 
 // Constants ----------------------------------------------
 
+const usingTestchain = !!getUrlParam('testchain_id');
+const CHIEF = usingTestchain ? 'MCD_ADM' : 'CHIEF';
```

Comment:
wait, this should be toggled based on `useMcdKovanContracts`, not `usingTestchain`.